### ISSUE TITLE
Get report rules validation from Sandra

### ISSUE THREAD

**Issue by baudoliver7** [endeavourmining/report-bot#48]
We have to got from Sandra a list of validation rules for reports.

**Comment by baudoliver7** [comment]
Cc Sandra - mail of  Friday 22/07/2022 08:02

Hi Olivier,

See below the validation rules.

Rules for successful import:
- Every template needs to be identical. The data will change, but the row titles and column titles, and the number of rows and columns, cannot change. Any edits that the sites made will result in failed upload. Most common edits:
- Adding extra rows
- Deleting rows (rows that are inapplicable to the site should have blank data, not get deleted)
- Changing the row names/titles of (in columns D and E)
- Deleting columns (sometimes they delete blank columns, or delete columns from previous dates)
- Changing the column titles (the dates) (rows 4 and 5). For example, if they do not want to enter data for months or weeks (which is how the template is set up), they will change the date ranges to match their own scheduling date ranges).
- Basically the row and columns titles need to be identical and in the same place

Rules for data validation:
- Rows with % in UOM column should only have data in the form 50% or 0.5. If they write the number 50, the dashboard will display 5000% since the dashboard format is set to %. This causes a lot of problems.
- A user will type a comma for a decimal place, even though their Excel setting is period for decimal places. Then the dashboard will read a number like 2,56 (2.56) like 2560. This is mostly a problem for grades. I think there should be a rule that any row with grade (UOM = g/t) should be < 20

Thank you again for your help,
Sandra
